packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -77,11 +81,19 @@ const FolderTreeItem = ({
   ) : null;
 
   const itemIcon = subfolder ? <IconFolder a11yLabel={intl.formatMessage({ id: 'Terra.folder-tree.folder-icon' })} /> : icon;
+  const expandCollapseIcon = isExpanded
+    ? <IconCaretDown height="8px" width="8px" />
+    : <IconCaretRight height="8px" width="8px" />;
```

Comment:
Is there a requirement to reduce the icon size?

Reply:
Yes, the expand/collapse icon should be 8px x 8px per UX requirements.